src/components/CodeReminderModal/index.jsx

```diff
@@ -123,6 +126,11 @@ class CodeReminderModal extends React.Component {
       template_closing: formData['email-template-closing'],
     };
 
+    // If the enterprise has a learner portal, we should direct users to it in our assignment email
+    if (enableLearnerPortal && configuration.ENTERPRISE_LEARNER_PORTAL_URL) {
```

Comment:
Nit: Since I think the backend does the default can you add a comment here about what the default is